hash/vsmile_cart.xml

```diff
@@ -1420,7 +1455,22 @@ V.Smile Smartbook Smartidges (need a Smartbook touch tablet connected to a regul
 		</part>
 	</software>
 
-	<software name="cinderla">
+	<software name="cinderlar101">
```

Comment:
Usually we use the shortest name for the latest revision and include the revision in the names for the older revisions.  Do you know what the revision number is for the other set?